### ISSUE TITLE
Training loss suddenly drops after a while of training

### ISSUE THREAD

**Issue by Singularity0104** [facebookresearch/vjepa2#74]
I'm reproducing vjepa2 pre-training using a custom dataset and the settings in https://github.com/facebookresearch/vjepa2/blob/main/configs/train/vitg16/pretrain-256px-16f.yaml. I'm observing a trend in training loss that initially decreases and then increases, consistent with the description in https://github.com/facebookresearch/vjepa2/issues/56. However, as training continues, I'm observing a sudden increase and then a decrease in the loss curve. Has this phenomenon occurred in your training? Does this indicate model pre-training failure?

<img width="800" height="500" alt="Image" src="https://github.com/user-attachments/assets/e6cf4535-d2e8-40f5-aeeb-dd09fea83824" />

**Comment by ballasnicolas** [comment]
Hi @Singularity0104,

Yes this is usually a sign of partial collapse that might hurts the downstream task performances. Usually, we stabilize training by exploring various LR/EMA parameters.

**Comment by Singularity0104** [comment]
@mmuckley @ballasnicolas I continued training the model and found that after the loss stabilized, the loss crash occurred more frequently, almost every 3000-5000 steps. Have you ever encountered this situation? How did you solve it?

<img width="741" height="465" alt="Image" src="https://github.com/user-attachments/assets/bcca77d6-58e5-455b-badd-083ae29b20e5" />
